tests/test_dataset_broker.py

```diff
@@ -7,37 +7,66 @@
 
 import pytest
 import os.path
+import re
 import time
-from subprocess import Popen
-from subprocess import call
+import tempfile
 import signal
 
+from subprocess import call, Popen
+
 producer_path = "../build/tests/boost/dataset_broker_producer"
 producer2_path = "../build/tests/boost/dataset_broker_producer2"
 consumer_path = "../build/tests/boost/dataset_broker_consumer"
-broker_path = "../build/ext/src/ch_acq/dataset_broker.py"
 
 
 def test_produce_consume():
+    broker_path = [
```

Comment:
Can't this just find `comet` in the current `PATH`. If you're willing to restrict to Python 3, then `shutil.which("comet")` will do that.

Reply:
Thanks, that's nice.